GpxLocationManager/Extensions/CLLocation.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  CLLocation+Utility.swift
+//  Eunomia
+//
+//  Created by Ian Grossberg on 8/24/18.
+//  Copyright © 2018 Adorkable. All rights reserved.
```

Comment:
That is fine that you retain the copyright, but there is a question as to whether `GpxLocationManager`'s license applies to files you wrote. Could you add the following sentence to the end of `LICENSE`: This license applies to all files in this repo, regardless of author.